flexget/components/sites/sites/serienjunkies.py

```diff
@@ -107,43 +179,82 @@ def parse_downloads(self, series_url, search_title):
             if not episode_title:
                 continue
 
-            # find download container
-            episode = episode_title.parent
-            if not episode:
-                continue
+            urls = self.find_urls(urls, episode_title)
+
+        if len(urls) == 0 and season_pack_fallback:
```

Comment:
more pythonic to do this:
```suggestion
        if not urls and season_pack_fallback:
```